modules/terminate-workers/variables.tf

```diff
@@ -0,0 +1,71 @@
+# ----------------------------------------------------------------------------
+# Terminate Workers - Input Variables
+# ----------------------------------------------------------------------------
+variable "name" {
+  description = "The name of the Lambda function to create."
+  type        = string
+}
+
+variable "asg_name" {
+  description = "The name of the Auto Scaling Group to attach to."
+  type        = string
+}
+
+variable "asg_arn" {
+  description = "The ARN of the Auto Scaling Group to attach to."
+  type        = string
+}
+
+variable "create" {
```

Comment:
question: I am not 100% sure how to use this variable. What is the use case not to create the resources? This results in a module which does not the work it was written for, right? Wouldn't I get the same result in not creating the whole module as every resource has a `count = var.create ? 1 :0`?

Reply:
You're right - this is useless in the submodule. I've removed it in a290efa4934c4a43c3f14143be6f21d5ba1ef689